l2geth/core/types/transaction_meta.go

```diff
@@ -160,14 +160,10 @@ func TxMetaEncode(meta *TransactionMeta) []byte {
 		common.WriteVarBytes(b, 0, l.Bytes())
 	}
 
-	queueOrigin := meta.QueueOrigin
-	if queueOrigin == nil {
```

Comment:
The queue origin should never be nil anywhere but this is technically a breaking change to the db. I'd like to upgrade my replica to this code to be sure that it just works

Reply:
We really need to add a replica test that we can trigger remotely

Reply:
I used this code on top of a node that was synced with the old code and confirmed this is backwards compatible under the reasonable assumption that the queue origin is always defined